library/ui-styles/src/main/res/values/palette.xml

```diff
@@ -41,26 +41,17 @@
     <color name="palette_tchap_light_blue">#2F80ED</color>
 
     <!-- For all themes Tchap -->
-    <color name="palette_tchap_dark_blue">#162D58</color>
-    <color name="palette_tchap_dark_blue_alpha25">#3F162D58</color>
-    <color name="palette_tchap_dark_grey_blue">#374C72</color>
+    <color name="palette_tchap_official_blue">#000091</color>
+    <color name="palette_tchap_official_white">#FFFFFF</color>
+    <color name="palette_tchap_official_red">#E1000F</color>
+    <color name="palette_tchap_official_black">#232323</color>
```

Comment:
@giomfo are you agree with the naming of these colors ? Do we need update colors (ids and values) in Tchap iOS ?

Reply:
@giomfo please add an issue to update colors ids and values in tchap ios